src/CHIPDriver.hh

```diff
@@ -55,7 +55,7 @@ void CHIPInitializeCallOnce(std::string BE = "");
  */
 void CHIPUninitializeCallOnce();
 
-std::string read_env_var(std::string ENV_VAR);
+std::string read_env_var(std::string ENV_VAR, bool lower);
```

Comment:
I wouldn't add a setting for 'lower', but just add an additional to_lower() std::string helper function.

Reply:
> additional to_lower() std::string helper function 
Add it where?

Reply:
To a new Utils.cpp or such module, or similar?

Reply:
ok I see, I will move it to utils if I ever need need to use it again in the future. Since env vars is the only place where I need it right now I will keep it as is.